Assets/link.xml

```diff
@@ -0,0 +1,3 @@
+<linker>
```

Comment:
What's this file for? Can it live inside any MRTK folder, or does it have to be at the Assets root?

Reply:
i can delete this. its auto generated for android if i remember correctly